tracer/build/_build/NukeExtensions/DockerDependencyType.cs

```diff
@@ -14,4 +14,9 @@
     /// Requires Docker on all platforms except Windows to run samples.
     /// </summary>
     LinuxAndMac,
+
+    /// <summary>
+    /// Contains both tests that require Docker and tests that do not.
+    /// </summary>
+    Mixed,
```

Comment:
FYI, I'm working on a PR currently that moves the build to a completely separate stage, and just builds _all_ the samples. That should both speed up the build, and remove some of this complexity. Once I've done that, I think we can just ditch this entirely. It's all fine for now though 👍